Problem140_LabelPartitioning.java

```diff
@@ -0,0 +1,37 @@
+//Time Complexity: O(n)
+//Space Complexity: O(n)
```

Comment:
Space is O(1) as we are using fixed size map or array

Reply:
Oh yes! forgot about the "fixed" size. Thanks for pointing out the correction.